data/data-pipeline/data_pipeline/etl/sources/eamlis/etl.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+import geopandas as gpd
+import pandas as pd
+from data_pipeline.config import settings
+
+from data_pipeline.etl.base import ExtractTransformLoad, ValidGeoLevel
+from data_pipeline.etl.sources.geo_utils import add_tracts_for_geometries
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class AbandonedMineLandInventorySystem(ExtractTransformLoad):
```

Comment:
I think our convention has been "AbandonedMineETL" or something similar, but I don't have strong feelings?

Reply:
Thank you!

Reply:
That is a less silly name: 702804e39047697fd3673a64fdd56256dfd82d58